esp/tools/soapplus/http.cpp

```diff
@@ -1411,17 +1411,16 @@ int HttpClient::validate(StringBuffer& xml)
         try
         {
             Owned<IPropertyTree> testTree = createPTreeFromXMLString(len, bptr, ipt_caseInsensitive);
-            fprintf(m_logfile, "Successfully parsed XML\n");
+            if ( (!m_doStress) || (m_doStress && http_tracelevel >= 5) )
+                fprintf(m_logfile, "Successfully parsed XML\n");
```

Comment:
unrelated to Windows build error, but took opportunity to add finer control of printing validation with --stress option for better output.